md/release-notes.md

```diff
@@ -7,3 +14,15 @@
  
  Only in the case you have created custom widgets using this silent library, you need to update them by adding UI Bootstrap js as an asset.
 
+
+### Full Java 8 support
+Bonita BPM 7.5.0+ is now fully Java 8 compliant.  
+Java 8 syntax and specific APIs can now be fully used in Connectors, Actor filters, Groovy Scripts, Custom REST APIs, Custom Event Handlers ...  
```

Comment:
We should not have "etc" in this sentence as it does not help the user to understand wether he/she can use Java 8 when implementing an extension point.
Either list all the extension point or state that all Extension Points can now be implemented in Java 8 (This will avoid to have to maintain the list accurate).
Can we link to the page of stable extension points?